### PR TITLE
CON-1521 migration from master to main

### PR DESCRIPTION
**Pull request by gyss**
`master` to `main` migration

I disabled the Firefox / Safari Karma tests. They are failing for more than one year ago and since this repo is unmaintained and the solution for this issue is not straightforward it makes sense to disable them for now

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -88,12 +88,20 @@ module.exports = function (karma) {
...
 				browser_version: 'latest',
 				os: 'Windows',
-				os_version: '10'
+				os_version: '10',
+				prefs: {
+					'privacy.trackingprotection.enabled': false,
+					'dom.serviceWorkers.enabled': true,
+					'security.OCSP.enabled': false,
+					'security.ssl.enable_ocsp_stapling': false
+				}
```
Comment (fenglish) [inline karma.conf.js:104]: If we are going to disable these tests, why is `prefs` added?

**Review by fenglish — COMMENTED** [review]
Looks good one comment

File: karma.conf.js (hunk 1 above)
Comment (gyss) [reply karma.conf.js:104]: I just wanted to note that Firefox needs to setup the security settings to make it work

but anyways I'll delete it

**Review by fenglish — APPROVED** [review]
Looks good to me

**Pull request merged**